flocker/node/agents/blockdevice.py

```diff
@@ -66,6 +66,10 @@ class DatasetStates(Names):
     """
     # Doesn't exist yet.
     NON_EXISTENT = NamedConstant()
+    # Existing volume not recorded as owning a dataset
+    UNREGISTGERED = NamedConstant()
```

Comment:
The explanatory comments are a little hard for me to understand, maybe clarify more what they mean?